### DISCUSSION TITLE
Model observations to actual observations

### DISCUSSION THREAD

**Discussion by kes79** [gmdsi/GMDSI_notebooks#107] [Q&A] [answered]
I'm working through the notebooks, but adjusting for the model I am working on...

In this case, the actual data I have is far more sparse than the modelled head obs csv output file. So in this case, I started with the observations as modelled observations (to check phi was zero) following 'part2_01_pstfrom_pest_setup', but then moving on to the tutorial notebook 'part2_02_obs_and_weights' I could update those with actual observation values, but now how do I remove all the observations that I do not have data for, and re-write the observation data?

**Comment by kes79** [comment, marked as answer]
Sorry, this could be deleted, I missed the crucial step of assigning all observation weights to zero, then only changing the weights for the obs I have data for

**Reply by rhugman** [reply]
Right on :) Keeping the discussion here for future reference.
